### ISSUE TITLE
Download progress invisible in Ubuntu (or Chromium?)

### ISSUE THREAD

**Issue by PropGit** [parallaxinc/Parallax-IDE#226]
The download progress bar is never visible when downloading on Ubuntu 14.04.  I'm using the default Chromium installation that came with the OS.

**Comment by phated** [comment]
It would be helpful if you can post the version reported by the browser.

**Comment by PropGit** [comment]
Good idea.  About Chromium says: Version 43.0.2357.130 Ubuntu 14.04 (64-bit)
